boxoffice/views/forms.py

```diff
@@ -0,0 +1,32 @@
+from wtforms import Form
```

Comment:
Still using `Form` from `wtforms` because of the [CSRF validation] requirement (https://github.com/hasgeek/baseframe/issues/110) in baseframe's `Form`.